### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by engrabdullahriaz**
In this PR, I implemented the following steps:

    1. I learned how to validate the form.
    2. Write a good readme file for a good understanding of the project by the user.
    3. Understanding the importance of the User Experience.

### CODE REVIEW

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi Team,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [x] Next time please implement all your change inside the branch you making the pull request with so the reviewer can make a proper review with inline comments.

- [x] For the contact form please add the correct margin top for your section header and add the top left curvature and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image  | image |

- [x] Kindly add hover and active (pressed) button interactions style to match the Figma design.

image

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 1]
```diff
@@ -3,17 +3,21 @@
...
 ![screenshot](./imgg.png)
 
 ## Built With
 
-- HTML/CSS 
-- Linters 
+- HTML/CSS
+- Linters
 
 ## Live Demo (if available)
```
Comment (arslanbisharat) [inline README.md:23]: - [x] [OPTIONAL] Please add a proper live link to your readme.

File: index.html [hunk 2]
```diff
@@ -282,6 +282,7 @@ <h4>LET'S CONNECT</h4>
       </section>
 
       <section class="contact-me">
```
Comment (arslanbisharat) [inline index.html:284]: - [x] As mentioned by the previous reviewer, please make the changes so your design looks like Figma's design. Your design looks like this right now. 

image

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html [hunk 3]
```diff
@@ -282,6 +282,7 @@ <h4>LET'S CONNECT</h4>
       </section>
 
       <section class="contact-me">
+        <br />
         <h6>Contact me</h6>
         <p>
           If you have an application you are interested in developing, a feature
```
Comment (prolajumokeoni) [inline index.html:288]: ### OPTIONAL

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
Screenshot_181  | Screenshot_180

- [x] Good job making all the requested changes 👍🏿  Kindly increase the with of the paragraph text to have the same width as the form
- [x] Please change the  color for name and email inputs to the color used in the Figma design 👍🏿

**Review by prolajumokeoni — APPROVED** [review]
Hi @Team

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀 
- Good design 💯  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Check inline comments_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**